src/redux/reducers/rootReducer.js

```diff
@@ -0,0 +1,10 @@
+/*
+* Here will be added all the created reducers
+* and combine all of them using combinedReducers
+*/
+import { combineReducers } from 'redux';
+import welcome from './index';
```

Comment:
Did you know that when importing from `index.js` may not require `./index`. 
You can just do :
```javascript 
import welcome from './'
```
I thought you can get familiar with it.